charts/gluu-all-in-one/values.yaml

```diff
@@ -18,6 +18,21 @@ state: TX
 cnPersistenceType: sql
 # -- Salt. Used for encoding/decoding sensitive data. If omitted or set to empty string, the value will be self-generated. Otherwise, a 24 alphanumeric characters are allowed as its value.
 salt: ""
+# -- service account used by Kubernetes resources
+serviceAccountName: default
```

Comment:
this shouldn’t be `default` since a SA resource has been added . This will cause a conflict with the `default` SA provided by the kubernetes cluster.

Reply:
There's  an if condition that will avoid the conflict in serviceaccount.yaml 
`{{- if not (eq .Values.serviceAccountName "default") -}}`